source/backtraceClient.ts

```diff
@@ -33,6 +38,20 @@ export class BacktraceClient extends EventEmitter {
     this._clientRateLimit = new ClientRateLimit(this.options.rateLimit);
     this.registerHandlers();
     this.setupScopedAttributes();
+
+    this.attributes = this.getClientAttributes();
+    if (this.options.enableMetricsSupport) {
+      this._backtraceMetrics = new BacktraceMetrics((clientOptions as BacktraceClientOptions), () => {
+        return this.getClientAttributes();
+      });
+    }
+  }
+
+  private getClientAttributes() {
+    return {
+      ...readSystemAttributes(),
```

Comment:
are you sure we should read dynamic attributes (like memory usage - attribute that changes over the time) and pass them to metrics support?

Reply:
We don't pass the return value of `getClientAttributes`, we pass a function `() => this.getClientAttributes()` so it refetches the attributes every time it is called, so they will update dynamically